doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -1717,7 +1740,7 @@ This rule will deny all requests for URIs with the ``.php`` file extension::
 Use GMT regardless of system timezone setting
 ---------------------------------------------
 
-This rule will change the behavior of %{NOW}. It will always return time in GMT.
+This rule will change the behavior of %{NOW}. It will always return time in GMT.::
```

Comment:
This renders as `.:`. I think we want `::`. Or, does `.:` work? I don't know offhand how to address this in Sphinx.

I see now that there is a lot of `.::` in this file. Can you fix those as well. :)